### ISSUE TITLE
I want to add Chinese translation to the nhentai extension, what should I do?

### ISSUE THREAD

**Issue by ZerOri** [tachiyomiorg/tachiyomi-extensions#9446]
### Describe your suggested feature

Text of the filter in extension.
![QQ图片20211012155329](https://user-images.githubusercontent.com/46775615/136914997-59e56b1d-d39a-479f-8c5d-5d9482461f91.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I will fill out all of the requested information in this form.

**Comment by alessandrojean** [comment]
There's a current limitation with the extensions that won't let you translate in the normal way apps do (using resources). Any other implementation would depend on adding every single string to the class, and it would be ugly and not really functional.

Tl;dr; you can't translate these at the moment.
